packages/react-components/src/components/styles/DomainPermissions.styles.ts

```diff
@@ -9,11 +9,23 @@ import { _pxToRem } from '@internal/acs-ui-common';
  */
 export const iconContainerStyles: IStackStyles = {
   root: {
-    margin: 'auto',
+    marginTop: 'auto',
+    marginBottom: 'auto',
     position: 'relative'
   }
 };
 
+/**
+ * @internal
+ */
+export const iconBannerContainerStyles: IStackStyles = {
+  root: {
+    paddingBottom: '1rem',
+    width: '20rem',
```

Comment:
is width and margin needed or can we remove? not clear what they're doing here

Reply:
(I would expect width to be `100%` if its needed)

Reply:
not needed so removed.